### ISSUE TITLE
Verbose CLI output

### ISSUE THREAD

**Issue by justenh** [putyourlightson/craft-blitz#642]
### Feature Request

Provide more detailed output during CLI execution. 

For example, running `craft blitz/cache/generate-site 1`, it would be great to see which pages are being processed at any given time. This would have the added benefit of providing a way to troubleshoot particularly lengthy generations, or exceptions that cause the command to fail. Ideally, these would be written directly to the CLI during execution so our deployment scripts would capture them as the output of the call. Below is a rough outline of the type of information that might be useful. 

```
Generating Blitz cache for 3 uris.
1. Updated cache "/newsroom/page" (500ms)
2. Updated cache "/newsroom/page-2" (1500ms)
3. Removed cache "/newsroom/page-3" (100ms)

Generated Blitz cache for 18/18 uris
2 updates
1 removals
total time: 2m10s
```

Additionally, running any CLI that combines multiple commands such as `blitz/cache/refresh` could output each step of the command being executed. Currently, the `refresh` command seems to hang with no output.

If the output would cause a slowdown in the execution of the command, it could be flagged on and off. Our preference would be not to hide it behind debug, but instead have it accept a flag of `--verbose` or something similar.

**Comment by bencroker** [comment]
Thanks for the suggestion. I added a verbose output mode to `blitz/cache` console commands in https://github.com/putyourlightson/craft-blitz/commit/097dfc06fb28063eef5d327210e7fe75fee5dd79 that can be activated by adding a `--verbose` flag. It doesn’t quite report everything in your outline, but it lists URLs as they are generated, noting any errors in red.

You can test this by running `ddev composer require "putyourlightson/craft-blitz:dev-develop as 4.14.0"`.

**Comment by justenh** [comment]
Thanks @bencroker! I Just ran through a quick test, and the output is very helpful. I'll continue to test and let you know if anything pops up. Appreciate your quick attention to this request!

**Comment by bencroker** [comment]
Released in [4.14.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.14.0).

**Comment by ammannbe** [comment]
Does that `--verbose` flag still exist in Version 5?

I get following output when I leave `--verbose` away or set it to `0`:

```bash
# php craft blitz/cache/refresh --verbose 0
Blitz cache successfully cleared.
Blitz cache successfully flushed.
Cache purging is disabled.
Generating Blitz cache...
[==================================================================================================================================================================================================] 100% (333/333) ETA: 0 sec.
Blitz cache generation complete.
Deploying is disabled.
```

We normally use a cronjob this command which still throws a message to stderr (which IMO is not an error but just an info):
```bash
# php craft blitz/cache/refresh > /dev/null
Cache purging is disabled.
Deploying is disabled.
```

Is it possible to avoid any non-error outputs?

**Comment by bencroker** [comment]
@ammannbe fixed in https://github.com/putyourlightson/craft-blitz/commit/16d513db47e1d0933ffba209cb4c70ef60157847 for the next release.

**Comment by bencroker** [comment]
Released in [5.12.4](https://github.com/putyourlightson/craft-blitz/releases/tag/5.12.4).
